public/app/features/alerting/unified/components/receivers/form/GrafanaReceiverForm.test.tsx

```diff
@@ -166,6 +177,14 @@ describe('GrafanaReceiverForm', () => {
   });
 });
 
+function renderForm(component: ReactNode) {
```

Comment:
not a blocker but I think we could use `renderWithProvider` util function instead?